src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1583,6 +1583,7 @@ internal enum ErrorCode
         ERR_FeatureNotAvailableInVersion8 = 8400,
         ERR_AltInterpolatedVerbatimStringsNotAvailable = 8401,
         WRN_DefaultLiteralConvertedToNullIsNotIntended = 8402,
+        ERR_UnconstrainedTypeParameterInNullCoalescingNotAvailable = 8403,
```

Comment:
>ERR_UnconstrainedTypeParameterInNullCoalescingNotAvailable [](start = 8, length = 58)

It looks like this error code is not used and that matches what I would expect. Remove?